spec/app/domain/util/log_message_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'Util::LogMessageClass' do
+  context 'object' do
+    let(:log_message) { "some log message" }
+    subject(:log_message_class) { Util::LogMessageClass.new(log_message) }
+
+    it 'has the expected message' do
+      expect(log_message_class.new.to_s).to eq(log_message)
+    end
+  end
+
+  context 'templating' do
+    let(:log_message_template) { "Variable is \#{0}." }
```

Comment:
small point: can use single quotes to avoid the backslash.  you might consider using a character other than `#` too, because at first I thought it was part of the required template string, but then I realized you were just using it as a marker to you could test more clearly against nil and empty strings.

Reply:
good to know, thanks!

The # was actually from the existing error_class.spec so i removed it from there too